src/members/mrdemonwolf.md

```diff
@@ -1,4 +1,19 @@
 ---
 username: mrdemonwolf
 profile: ../img/members/mrdemonwolf.png
+twitter: mrdemonwolf
+instagram: mrdemonwolf
+github: mrdemonwolf
+youtube: mrdemonwolf
```

Comment:
```suggestion
youtube: https://www.youtube.com/mrdemonwolf
```

The youtube field has to be a full URL as not everyone has a custom URL. I'll commit this for you and then merge this in